dask/dataframe/io/parquet.py

```diff
@@ -74,6 +74,11 @@ def _read_fastparquet(fs, paths, myopen, columns=None, filters=None,
     elif len(minmax) > 1:
         if index:
             index_col = index
+        elif index is None:
+            # attempt auto-index
+            index = pf._get_index()
+            if index:
+                index_col = index
         elif 'index' in minmax:
             index_col = 'index'
```

Comment:
A bit magic, perhaps, but with potentially great performance implications - so the usual trade-off. In addition, this might happen to be the right thing to do in many cases, where the data has come from some other table-based data system. I don't feel qualified to guess what the user might guess as reasonable, I would say the most important thing is to keep the documentation very clear.